spec/cobra_commander/cli_spec.rb

```diff
@@ -41,15 +41,21 @@
         │               └── f
         └── node_manifest
             ├── b
-        │       └── g
-        │           ├── e
-        │           └── f
+            │   └── g
+            │       ├── e
+            │       └── f
             ├── e
             ├── f
             └── g
                 ├── e
                 └── f
       OUTPUT
+
+      # This converts a unicode non-breaking space with
+      # a normal space because editors.
+      expected_output = expected_output.strip.tr("\u00a0", " ")
```

Comment:
It wasn't straightforward to figure out, but `\u00a0` is the unicode value of 160, a non-breaking space, which utf-8 documents will use as a default for a normal space character. That *should* be character 32, but for some reason is not, I assume in the spec.

Since a space is a space, to get the spec to pass, I added the code which replaces that specific character, 160, so the spec would ignore the variance.

Reply:
Nice work! Thank you!